vcbuild.bat

```diff
@@ -174,6 +174,20 @@ if defined package set stage_package=1
 
 :: assign path to node_exe
 set "node_exe=%config%\node.exe"
+if "%target_arch%"=="arm64" if "%PROCESSOR_ARCHITECTURE%"=="AMD64" (
+  if not defined "%x64_node_exe%" set "x64_node_exe=temp-vcbuild\node-x64-cross-compiling.exe"
+  if not exist "%x64_node_exe%" (
+    echo Downloading x64 node.exe...
+    if not exist "temp-vcbuild" mkdir temp-vcbuild
+    powershell -c "Invoke-WebRequest -Uri 'https://nodejs.org/dist/latest/win-x64/node.exe' -OutFile 'temp-vcbuild\node-x64-cross-compiling.exe'"
+  )
+  if not exist "%x64_node_exe%" (
+    echo Could not find the Node executable at the given x64_node_exe path. Aborting. 
+    goto exit
+  )
+  echo Using x64 Node executable because we're cross-compiling for arm64: %x64_node_exe%
+  set "node_exe=%x64_node_exe%"
```

Comment:
`node_exe` should always point to the binary being built. Linters, node-gyp, npm, all of that should fail when cross-compiling.

Please use a dedicated variable name, and use it only to run things that are absolutely needed to build a release (only licence2rtf?).

Reply:
Fixed 👍